extensions.html

```diff
@@ -335,6 +345,57 @@ <h2><dfn>GamepadPose</dfn> Interface</h2>
       </dl>
     </section>
 
+    <section>
+      <h2><dfn>GamepadTouch</dfn> Interface</h2>
+      <p>
+        This interface defines a single touch event on a gamepad device that
+        supports input. The event consists of a touch id that uniquely
+        identifies the touch point from the time the input medium (e.g. finger,
+        stylus, etc) makes contact with the touch device, up to the time the
+        input medium is no longer making contact with the touch device.
+      </p>
+    
+      <pre class="idl">
+        [Exposed=Window, SecureContext]
+        interface GamepadTouch {
+          readonly attribute unsigned long touchId;
+          readonly attribute octet surfaceId;
+          readonly attribute Float32Array position;
+          readonly attribute Uint32Array? surfaceDimensions;
```

Comment:
Why not just an empty array?

Reply:
Hi @marcoscaceres we thought about representing the absent surfaceDimensions with "null". Would there be any advantages to use the empty array instead? thanks~